rest_framework_bulk/serializers.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from rest_framework.serializers import ListSerializer
+
+
+class BulkSerializerMixin(object):
+    def to_internal_value(self, data):
+        ret = super(BulkSerializerMixin, self).to_internal_value(data)
+        if isinstance(self.root, BulkListSerializer) and self.context['view'].action in ('partial_bulk_update', 'bulk_update'):
```

Comment:
how did this work for you in regular views? `view.action` only seems to be set on viewsets - https://github.com/tomchristie/django-rest-framework/blob/master/rest_framework/viewsets.py#L107